components/operator/Earthfile

```diff
@@ -74,6 +74,10 @@ helm-update:
     SAVE ARTIFACT helm AS LOCAL helm
 
 deploy:
+    ARG LICENCE_TOKEN=""
+    ARG LICENCE_CLUSTER_ID=""
+    ARG LICENCE_ISSUER="http://licence-api.formance.svc.cluster.local:8080/keys"
```

Comment:
You can retrieve the uri with licence deployment

Reply:
This is the default value, we can override this one by calling the + deploy with a flag value